demo

```diff
@@ -52,7 +59,7 @@ done
 # Check if the demo-Artifact has been downloaded,
 # or if there exists a newer one in storage.
 DEMO_ARTIFACT_NAME="mender-demo-artifact.mender"
-wget -q --timestamping https://dgsbl4vditpls.cloudfront.net/${DEMO_ARTIFACT_NAME}
+wget -q https://dgsbl4vditpls.cloudfront.net/${DEMO_ARTIFACT_NAME}
```

Comment:
Careful.

Without specifying the `-O name` a new version of the Artifact will never be downloaded